subiquity/models/subiquity.py

```diff
@@ -159,6 +159,10 @@ def __init__(self):
         self.selections = ''
 
     def render(self):
+        return {}
```

Comment:
So for a while we've been moving away from the binary install/postinstall model distinction and this just moves it further. No action required for this PR -- but I guess this method name is now too generic and we should at some point change it, or change it to something that takes an argument or something.

Reply:
Agreed, render() and serialize() are the two serialization functions that I always mix up because of how vague their name are.

Currently, we have:
* `model.make_cloudconfig`
* `model.render` (targets curtin install)
* `model.get_apt_config` (targets curtin apt-config)

* `controller.make_autoinstall`
* `controller.serialize` (for the refresh / restart)

IMO the ones that start with `make_` have better names.

Reply:
Yeah serialize is far too generic a name for the specific and obscure thing those methods do!